Polygon/include/CGAL/Polygon_2/Polygon_2_impl.h

```diff
@@ -94,7 +94,7 @@ operator>>(std::istream &is, Polygon_2<Traits_P,Container_P>& p)
 {
   int n = 0; // number of vertices
   is >> n;
-  typename Traits_P::Point_2 point;
+  typename Traits_P::Point_2 point(0,0);
```

Comment:
That initialization only hides the warning, but the compiler is right: there is a problem with the code. The real problem is below:
```c++
      for (int i=0; i<n; i++) {
        is >> point;
        p.push_back(point);
      }
```
It should be:
```c++
      for (int i=0; i<n; i++) {
        if(is >> point) {
          p.push_back(point);
        } else {
          return is;
        }
      }
```

Reply:
Ow ok.